test/unit/calculators/maternity_paternity_calculator_test.rb

```diff
@@ -483,7 +498,9 @@ class MaternityPaternityCalculatorTest < ActiveSupport::TestCase
         should "calculate pay due for each pay date on a weekly cycle" do
           @calculator.pay_date = Date.parse('28 December 2012') # Friday before maternity leave/pay starts.
           @calculator.pay_method = 'weekly'
-          @calculator.calculate_average_weekly_pay('weekly', 2000)
+          @calculator.pay_pattern = 'weekly'
+          @calculator.earnings_for_pay_period = 2000
+          @calculator.average_weekly_earnings
```

Comment:
Is this line (and a few others like it in this file) necessary now that `#average_weekly_earnings` doesn't change the state of the `@calculator`? They were changed in e6601288ac00cd7424c57acc8cb33be44706b803 ("Convert average_weekly_earnings into fully-fledged calculator method").